YooniK.Face/YooniK.Face.Client/Client/FaceClient.cs

```diff
@@ -138,20 +138,31 @@ public async Task<VerifyIdResponse> VerifyIdAsync(string template, string templa
         /// </summary>
         /// <param name="firstImage"> First base 64 string image. </param>
         /// <param name="secondImage"> Second base 64 string image. </param>
-        /// <returns><see cref="VerifyImagesResponse"/></returns>
-        public async Task<VerifyImagesResponse> VerifyImagesAsync(string firstImage, string secondImage)
+        /// <returns><see cref="MatchingResponse"/></returns>
+        public async Task<MatchingResponse> VerifyImagesAsync(string firstImage, string secondImage)
         {
             try
             {
-                var verifyImages = new VerifyImagesRequest(firstImage, secondImage);
-
-                var message = new RequestMessage(
-                    httpMethod: HttpMethod.Post,
-                    urlRelativePath: FaceEndpoints.VerifyImages,
-                    request: verifyImages
-                    );
-
-                return await _serviceClient.SendRequestAsync<VerifyImagesResponse>(message);
+                var firstProcess = ProcessAsync(firstImage);
+                var secondProcess = ProcessAsync(secondImage);
+
+                await Task.WhenAll(new [] { firstProcess, secondProcess });
+
+                if (firstProcess.Result.Count > 0 && secondProcess.Result.Count > 0)
```

Comment:
We should check each image individually include this information in the exception message. We should also check if more than one face is detected.